tests/System.Text.Formatting.Tests/Parsing/ParserTests.cs

```diff
@@ -0,0 +1,389 @@
+﻿using System.Runtime.InteropServices;
+using System.Text.Parsing;
+using Xunit;
+
+namespace System.Text.Formatting.Tests
+{
+    public class ParserTests
+    {
+        #region ulong
+        [Theory]
+        [InlineData(new byte[] { 0x31, 0x37, 0x32, 0x38 } /* "1728" */, true, 0, 1728, 4)]
```

Comment:
Feel free to do it as a separate commit, but it would be great to add some negative tests, i.e. tests that get false returned or exceptions thrown from TryParse methods.